src/lib/updateReasonNames.js

```diff
@@ -0,0 +1,81 @@
+import { Op } from 'sequelize';
+import { ActivityReport } from '../models';
+import { logger } from '../logger';
+
+/**
+ * put it in this object so we could add to this as necessary
+ */
+const REASON_DICTIONARY = [
+  {
+    old: 'Monitoring | Under Enrollment',
+    renamed: 'Full Enrollment',
+  },
+  {
+    old: 'Data Training or Analysis',
+    renamed: 'Ongoing Quality Improvement',
+  },
+  {
+    old: 'Professional Development',
+    renamed: 'Ongoing Quality Improvement',
+  },
+  {
+    old: 'Quality Improvement Support',
+    renamed: 'Ongoing Quality Improvement',
+  },
+  {
+    old: 'Program Planning',
+    renamed: 'Planning/Coordination (also TTA Plan Agreement)',
+  },
+  {
+    old: 'School Readiness',
```

Comment:
Looks like this and other reasons will need to be changed to add 'Need:' per TTHUB-263, e.g. 'Need: School Readiness'

Reply:
This has been updated.